res/values/strings.xml

```diff
@@ -465,6 +465,7 @@
     <string name="confirmation_remove_files_alert">Do you really want to remove the selected items?</string>
     <string name="confirmation_remove_folders_alert">Do you really want to remove the selected items and their contents?</string>
     <string name="maintenance_mode">Server in maintenance mode</string>
+    <string name="failed_upload_quota_exceeded_text">Not enough free space</string>
```

Comment:
We should specify in some way that it's a lack of space ** in the server **. I think it's important. Maybe

"Not enough free space in server", or "Not enough space in server" if the first is too long. 

cc @jesmrec for second opinion.